src/graphics/program-lib/chunks/skyboxHDRRot.frag

```diff
@@ -0,0 +1,11 @@
+varying vec3 vViewDir;
+
+uniform samplerCube texture_cubeMap;
+uniform mat3 cubeMapRotationMatrix;
+
+void main(void) {
+    vec3 color = processEnvironment($textureCubeSAMPLE(texture_cubeMap, fixSeamsStatic((vViewDir*vec3(-1,1,1))*cubeMapRotationMatrix, $FIXCONST)).rgb);
```

Comment:
Style (operators and commas)

Reply:
I'll do a spacing pass on all the shaders chunks I touched

Reply:
shader chunk spacing should all be fixed now - thanks!